drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -253,6 +254,14 @@ def addresses
       load_ar_association(object, :addresses)
     end
 
+    def hud_chronic
+      return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
+
+      # client.hud_chronic causes n+1 queries
+      enrollments = object.enrollments.hmis
```

Comment:
the `.hmis` really isn't necessary. `client.enrollments` will only return enrollments that are in the same data source as the client. but fine to leave it